os/os-windows.h

```diff
@@ -22,6 +22,14 @@
 
 #include "windows/posix.h"
 
+#ifdef _LP64
```

Comment:
I'm not sure introduction of these macros is the best solution. Since sizeof(DWORD) == sizeof(uint32_t), how about using uin32_t to store the result of GetLastError() instead of DWORD?

Reply:
I just use %lu types and some casting now.